assets/stats/lib/csv_link.es6

```diff
@@ -14,13 +14,28 @@ export class StatsCSVLink extends StatsUI {
   }
 
   template () {
-    let csvString = this.csvString
+    if (!this.csvString) {
+      return <span className="StatsCSVLink-disabled">Download CSV</span>
```

Comment:
Do we display "Download" if no data to download?

Reply:
It's rendered disabled and grayed. Otherwise _Loading CSV..._ or something like that could be misleading. It's only a failsafe case anyway.

Reply:
Why not just not show the link? Anyway, we can do that later.